Mage/src/main/java/mage/abilities/AbilitiesImpl.java

```diff
@@ -102,7 +102,7 @@ public AbilitiesImpl(final AbilitiesImpl<T> abilities) {
                     rules.add(sbRule.toString());
                 }
                 String rule = ability.getRule();
-                if (rule.length() > 0) {
```

Comment:
If we change this, we will probably get less likely aware of rule problems a card has.